internal/engine/filter/filter.go

```diff
@@ -50,7 +51,7 @@ func (f Filter) String() string {
 	w.WriteParam("from", f.Criteria.From)
 	w.WriteParam("to", f.Criteria.To)
 	w.WriteParam("subject", f.Criteria.Subject)
-	w.WriteParam("query", f.Criteria.Query)
+	w.WriteParam("query", "\n"+indent(f.Criteria.Query, 3))
```

Comment:
Why is the query always prefixed with a newline? I would expect everything to be in one line if there's no need for indentation.